### ISSUE TITLE
[BUG] "Repair Lemmy code blocks" fails to remove some <span> tags

### ISSUE THREAD

**Issue by pbackus** [aclist/kbin-kes#237]
**Describe the bug**

The "Repair Lemmy code blocks" mod should remove all of the `<span>` tags inserted into code blocks federated from Lemmy instances. However, some `<span>` tags are not removed.

**To Reproduce**

1. Go to <https://kbin.social/m/programming@programming.dev/t/726929/The-C-scope-resolution-operator-is-beautiful>
2. Enable "Repair Lemmy code blocks" in KES.
3. See that there are still `<span>` tags inserted by Lemmy in the displayed code block.

Without "Repair Lemmy code blocks" enabled, the first code block in the linked post looks like this:

```
<span style="color:#323232;">
</span><span style="color:#323232;"><span style="color:#0086b3;">Vec</span><span style="color:#323232;">::new();
</span>
```

With "Repair Lemmy code blocks" enabled, the first code block in the linked post looks like this:

```
<span style="color:#0086b3;">Vec</span><span style="color:#323232;">::new();
```

**Expected behavior**

No `<span>` tags should appear in the linked post's code blocks when "Repair Lemmy code blocks" is enabled.

**KES version:**
- 3.1.0

**Extension and version:**
- Extension: TamperMonkey
- Version number: 5.0.0

**Desktop (please complete the following information):**
 - OS: Debian Linux 12.4
 - Browser: Firefox
 - Browser version: 122.0b3

**Comment by aclist** [comment]
Thank you for the report. I have brought this to the attention of the mod's author and suggested a few possible workarounds.

**Comment by Pamasich** [comment]
I'll look into this over the next week.

This does look like it complicates things quite a bit.

**Comment by aclist** [comment]
Resolved in 3.2.0
